packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,188 @@
+    $thickness: var(--_iui-expandable-block-header-focus-thickness)
+  );
+  display: flex;
+  gap: var(--_iui-expandable-block-header-gap);
+  align-items: center;
+  padding: var(--_iui-expandable-block-header-padding);
+  cursor: pointer;
+  user-select: none;
+  color: var(--iui-color-text);
+  line-height: inherit;
+  font-family: inherit;
+  text-align: left;
+  background-color: var(--_iui-expandable-block-background-color);
+  border: var(--_iui-expandable-block-border);
+  border-radius: inherit;
+  border-bottom-left-radius: var(--_iui-expandable-block-header-bottom-border-radius, inherit);
+  border-bottom-right-radius: var(--_iui-expandable-block-header-bottom-border-radius, inherit);
+  transition: background-color var(--iui-duration-1) ease-out;
+  @media (forced-colors: active) {
+    --_iui-expandable-block-expander-icon-fill: CanvasText;
+  }
+
+  &:where(:hover) {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-hover);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-hover);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-hover);
+
+    background-color: var(--_iui-expandable-block-header-background-color-hover);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-expander-icon-fill: Highlight;
+      --_iui-expandable-block-header-caption-color: Highlight;
+      --_iui-expandable-block-header-label-color: Highlight;
+      border-color: Highlight;
+    }
+  }
+
+  &[disabled],
+  &:disabled,
+  &[aria-disabled='true'] {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-disabled);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-disabled);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-disabled);
+
+    cursor: not-allowed;
+    background-color: var(--_iui-expandable-block-header-background-color-disabled);
+    border-color: var(--iui-color-border-disabled);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-expander-icon-fill: GrayText;
+      --_iui-expandable-block-header-caption-color: GrayText;
+      --_iui-expandable-block-header-label-color: GrayText;
+      border-color: GrayText;
+    }
+  }
+}
+
+/// @arg $type Can be one of: expander, status
+@mixin iui-expandable-block-header-icons($type) {
+  @include iui-icon-style('m');
+
+  @if $type == 'expander' {
+    width: var(--_iui-expandable-block-expander-icon-size);
+    height: var(--_iui-expandable-block-expander-icon-size);
+    fill: var(--_iui-expandable-block-expander-icon-fill);
+    transform: var(--_iui-expandable-block-expander-icon-transform, none);
+    transition: fill var(--iui-duration-1) ease-out;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: fill var(--iui-duration-1) ease-out, transform var(--iui-duration-1) ease-out;
+    }
+  }
+
+  @if $type == 'status' {
+    @media (forced-colors: active) {
+      fill: CanvasText;
```

Comment:
In table, the status icons have color in the forced color mode too.

![image](https://user-images.githubusercontent.com/45748283/203625646-416632d3-c39a-4156-af73-330df01c6fe0.png)

But here, I think they are not colored. 

![image](https://user-images.githubusercontent.com/45748283/203625706-73028311-337d-4cd0-a620-dd9160821080.png)

Not sure which one is correct, but maybe both examples should follow the same status colors in forced colors mode.

Reply:
Removed so the status color shows in `forced-colors`, matching what we have in table.